capstone_project/src/main/webapp/pages/comments.jsp

```diff
@@ -15,42 +15,64 @@ limitations under the License.
 <%! BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService(); %>
 
 <!DOCTYPE html>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
 <script src="../scripts/comments.js"></script>
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
 <link rel="stylesheet" href="../styles/posts.css">
 <body onload="loadPosts(); loadSchools()">
-  <div id="sort-types">
-    <button onclick="sortPosts('new')">New</button>
-    <button onclick="sortPosts('top')">Top</button>
-    <button onclick="sortPosts('trending')">Trending</button>
+  <div class="row">
+    <div class="col s3" id="sidebar">
+    </div>
```

Comment:
Can this closing div tag go on the line above, since there is nothing inside the div?